common/test/keyboards/issue/6254/build.sh

```diff
@@ -6,7 +6,7 @@ set -u
 ## START STANDARD BUILD SCRIPT INCLUDE
```

Comment:
The issue folder name could be `6254-invalid-version-99` -- that way it's easy to remember which is which.